website/homepage/nginx.conf

```diff
@@ -3,44 +3,38 @@ server {
     server_name  localhost;
     absolute_redirect off;
 
-    #access_log  /var/log/nginx/host.access.log  main;
-
     location / {
         root   /usr/share/nginx/html;
         try_files $uri $uri/ $uri/index.html =404;
         index  index.html index.htm;
         autoindex on;
     }
 
-    error_page  404              /404.html;
+    location /introduction_to_blocks {
+        return 301 /quickstart;
```

Comment:
Is it possible to include the anchor link specifically to this section? May not be possible via nginx I'm not sure

Reply:
will try

Reply:
This isn't possible with nginx, basically the # tag is a browser thing and the server ignores it. https://stackoverflow.com/a/9794542

Reply:
Ok good to know!

Reply:
actually I was wrong, it is. Will fix those